Author: Andreas Beckmann <anbe@debian.org>
Description: import 470.xx legacy ID list from 510.* README.txt
 some models are missing from the 470.* README.txt
Bug-Debian: https://bugs.debian.org/1011245
Forwarded: not-needed

--- a/NVIDIA-Linux/README.txt
+++ b/NVIDIA-Linux/README.txt
@@ -9180,6 +9180,166 @@ A5. NVIDIA GRID GPUS
  GRID K520                                        118A            D
 
 
+
+A6. The 470.xx driver still supports the following set of GPUs:
+
+
+ NVIDIA GPU product                               Device PCI ID*  VDPAU features
+ -----------------------------------------------  --------------  --------------
+ GeForce GTX 650                                  0FC6            D
+ GeForce GT 740                                   0FC8            D
+ GeForce GT 730                                   0FC9            C
+ GeForce GT 755M                                  0FCD            D
+ GeForce GT 640M LE                               0FCE            C
+ GeForce GT 650M                                  0FD1            D
+ GeForce GT 640M                                  0FD2            D
+ GeForce GT 640M LE                               0FD2 1028 0595  C
+ GeForce GT 640M LE                               0FD2 1028 05B2  C
+ GeForce GT 640M LE                               0FD3            C
+ GeForce GTX 660M                                 0FD4            D
+ GeForce GT 650M                                  0FD5            D
+ GeForce GT 640M                                  0FD8            D
+ GeForce GT 645M                                  0FD9            D
+ GeForce GT 740M                                  0FDF            D
+ GeForce GTX 660M                                 0FE0            D
+ GeForce GT 730M                                  0FE1            D
+ GeForce GT 745M                                  0FE2            D
+ GeForce GT 745M                                  0FE3            D
+ GeForce GT 745A                                  0FE3 103C 2B16  D
+ GeForce GT 745A                                  0FE3 17AA 3675  D
+ GeForce GT 750M                                  0FE4            D
+ GeForce GT 750M                                  0FE9            D
+ GeForce GT 755M                                  0FEA            D
+ GeForce 710A                                     0FEC            C
+ GeForce 820M                                     0FED            C
+ GeForce 810M                                     0FEE            C
+ Quadro K1100M                                    0FF6            D
+ Quadro K500M                                     0FF8            D
+ Quadro K2000D                                    0FF9            D
+ Quadro K600                                      0FFA            D
+ Quadro K2000M                                    0FFB            D
+ Quadro K1000M                                    0FFC            D
+ NVS 510                                          0FFD            D
+ Quadro K2000                                     0FFE            D
+ Quadro 410                                       0FFF            D
+ GeForce GTX TITAN Z                              1001            D
+ GeForce GTX 780                                  1004            D
+ GeForce GTX TITAN                                1005            D
+ GeForce GTX 780                                  1007            D
+ GeForce GTX 780 Ti                               1008            D
+ GeForce GTX 780 Ti                               100A            D
+ GeForce GTX TITAN Black                          100C            D
+ Tesla K20Xm                                      1021            D
+ Tesla K20c                                       1022            D
+ Tesla K40m                                       1023            D
+ Tesla K40c                                       1024            D
+ Tesla K20s                                       1026            D
+ Tesla K40st                                      1027            D
+ Tesla K20m                                       1028            D
+ Tesla K40s                                       1029            D
+ Tesla K40t                                       102A            D
+ Tesla K80                                        102D            D
+ Quadro K6000                                     103A            D
+ Quadro K5200                                     103C            D
+ GeForce GTX 680                                  1180            D
+ GeForce GTX 660 Ti                               1183            D
+ GeForce GTX 770                                  1184            D
+ GeForce GTX 660                                  1185            D
+ GeForce GTX 760                                  1185 10DE 106F  D
+ GeForce GTX 760                                  1187            D
+ GeForce GTX 690                                  1188            D
+ GeForce GTX 670                                  1189            D
+ GeForce GTX 760 Ti OEM                           1189 10DE 1074  D
+ GRID K520                                        118A            D
+ GeForce GTX 760 (192-bit)                        118E            D
+ Tesla K10                                        118F            D
+ GeForce GTX 760 Ti OEM                           1193            D
+ Tesla K8                                         1194            D
+ GeForce GTX 660                                  1195            D
+ GeForce GTX 880M                                 1198            D
+ GeForce GTX 870M                                 1199            D
+ GeForce GTX 760                                  1199 1458 D001  D
+ GeForce GTX 860M                                 119A            D
+ GeForce GTX 775M                                 119D            D
+ GeForce GTX 780M                                 119E            D
+ GeForce GTX 780M                                 119F            D
+ GeForce GTX 680M                                 11A0            D
+ GeForce GTX 670MX                                11A1            D
+ GeForce GTX 675MX                                11A2            D
+ GeForce GTX 680MX                                11A3            D
+ GeForce GTX 675MX                                11A7            D
+ Quadro K4200                                     11B4            D
+ Quadro K3100M                                    11B6            D
+ Quadro K4100M                                    11B7            D
+ Quadro K5100M                                    11B8            D
+ Quadro K5000                                     11BA            D
+ Quadro K5000M                                    11BC            D
+ Quadro K4000M                                    11BD            D
+ Quadro K3000M                                    11BE            D
+ GeForce GTX 660                                  11C0            D
+ GeForce GTX 650 Ti BOOST                         11C2            D
+ GeForce GTX 650 Ti                               11C3            D
+ GeForce GTX 645                                  11C4            D
+ GeForce GT 740                                   11C5            D
+ GeForce GTX 650 Ti                               11C6            D
+ GeForce GTX 650                                  11C8            D
+ GeForce GT 740                                   11CB            D
+ GeForce GTX 770M                                 11E0            D
+ GeForce GTX 765M                                 11E1            D
+ GeForce GTX 765M                                 11E2            D
+ GeForce GTX 760M                                 11E3            D
+ GeForce GTX 760A                                 11E3 17AA 3683  D
+ Quadro K4000                                     11FA            D
+ Quadro K2100M                                    11FC            D
+ GeForce GT 635                                   1280            D
+ GeForce GT 710                                   1281            D
+ GeForce GT 640                                   1282            C
+ GeForce GT 630                                   1284            C
+ GeForce GT 720                                   1286            D
+ GeForce GT 730                                   1287            C
+ GeForce GT 720                                   1288            D
+ GeForce GT 710                                   1289            D
+ GeForce GT 710                                   128B            D
+ GeForce GT 730M                                  1290            D
+ GeForce GT 730A                                  1290 103C 2AFA  D
+ GeForce GT 735M                                  1291            D
+ GeForce GT 740M                                  1292            D
+ GeForce GT 740A                                  1292 17AA 3675  D
+ GeForce GT 740A                                  1292 17AA 367C  D
+ GeForce GT 740A                                  1292 17AA 3684  D
+ GeForce GT 730M                                  1293            D
+ GeForce 710M                                     1295            C
+ GeForce 710A                                     1295 103C 2B0D  C
+ GeForce 710A                                     1295 103C 2B0F  C
+ GeForce 810A                                     1295 103C 2B20  D
+ GeForce 810A                                     1295 103C 2B21  D
+ GeForce 805A                                     1295 17AA 367A  D
+ GeForce 710A                                     1295 17AA 367C  C
+ GeForce 825M                                     1296            D
+ GeForce GT 720M                                  1298            C
+ GeForce 920M                                     1299            D
+ GeForce 920A                                     1299 17AA 30BB  D
+ GeForce 920A                                     1299 17AA 30DA  D
+ GeForce 920A                                     1299 17AA 30DC  D
+ GeForce 920A                                     1299 17AA 30DD  D
+ GeForce 920A                                     1299 17AA 30DF  D
+ GeForce 920A                                     1299 17AA 3117  D
+ GeForce 920A                                     1299 17AA 361B  D
+ GeForce 920A                                     1299 17AA 362D  D
+ GeForce 920A                                     1299 17AA 362E  D
+ GeForce 920A                                     1299 17AA 3630  D
+ GeForce 920A                                     1299 17AA 3637  D
+ GeForce 920A                                     1299 17AA 369B  D
+ GeForce 920A                                     1299 17AA 36A7  D
+ GeForce 920A                                     1299 17AA 36AF  D
+ GeForce 920A                                     1299 17AA 36F0  D
+ GeForce GT 730                                   1299 1B0A 01C6  C
+ GeForce 910M                                     129A            D
+ Quadro K610M                                     12B9            D
+ Quadro K510M                                     12BA            D
+
+
+
 Below are the legacy GPUs that are no longer supported in the unified driver.
 These GPUs will continue to be maintained through the special legacy NVIDIA
 GPU driver releases.
